### ISSUE TITLE
Invalidate Result cache?

### ISSUE THREAD

**Issue by stephanvierkant** [stwe/DatatablesBundle#987]
I'm using the Result cache ([Resources/doc/cache.md](https://github.com/stwe/DatatablesBundle/blob/master/Resources/doc/cache.md)) but when I change the data, the cache doesn't invalidate. I'm using Docrtrine Cache on multiple places in my application and it works as expected, but for Datatables cache invalidation doesn't work.

Anyone found out how to invalidate the Doctrine cache that this library uses?

**Comment by Eloar** [comment]
How do you change data?

**Comment by Seb33300** [comment]
Yeah, doctrine cache only works when data is updated with doctrine.
If you update the data directly in the database, the cache will not be invalidated.

**Comment by stephanvierkant** [comment]
> How do you change data?

I'm changing the data with a Symfony Form
